src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const mainPhoto = document.querySelector('#largeImg');
```

Comment:
It's good practice to check if 'mainPhoto' actually found an element before trying to access its 'src' property to avoid potential runtime errors.